pkg/prom/ha/server.go

```diff
@@ -101,10 +101,12 @@ type Server struct {
 	exited chan bool
 
 	closeDependencies func() error
+
+	prometheusWriteConfig []*instance.RemoteWriteConfig
```

Comment:
Hmm, what do you think about making our own `GlobalConfig` struct in `pkg/prom/instance` and using that instead? I don't want to grow the list of params beyond what it is already:

```
type GlobalConfig struct {
  Prometheus *config.GlobalConfig 
  DefaultRemoteWrite []*RemoteWriteConfig
}
```

Reply:
That has a larger blast radius than expected when I tried it. I would prefer to get what I have into main and then do a separate PR for that.

Reply:
SGTM